src/renderer/html_handlebars/helpers/toc.rs

```diff
@@ -75,6 +75,7 @@ impl HelperDef for RenderToc {
                 ("", 1)
             };
 
+            #[warn(clippy::if_same_then_else)]
```

Comment:
I'm not sure why this is added.

Reply:
I added this to avoid having a error after executing `clippy`:
![Screenshot from 2020-05-10 10-17-35](https://user-images.githubusercontent.com/886331/81494243-a2fe7400-92a7-11ea-862d-d332030ce2b4.png)

Reply:
Ah, I see.  I just pushed a change to rewrite it: #1214.  I usually prefer unrelated changes like this to be separate.